lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -92,5 +90,6 @@ private function build($data) {
     $this->created_at = $data->timestamp;
```

Comment:
The `timestamp` property isn't available on the returned data and it's :hankey:ing on that. If you add `timestamps` as a field to grab info from in the query here:
```
$query->fields('s', ['sid', 'uid', 'nid', 'run_nid', 'timestamp']);
```
then all results should have the timestamp available for you to use ;)